docs/public/static/sponsors/canada-casino.svg

```diff
@@ -0,0 +1,9 @@
+  </defs>
+</svg>
```

Comment:
There stopped the sponsorship

```suggestion
```

Reply:
Should we add a "Become our sponsor" as there is now one spot left on the Gold grid?!

<img width="400" alt="Screen Shot 2021-09-06 at 12 08 03" src="https://user-images.githubusercontent.com/67129314/132236891-13186cfd-0b56-47b6-bfe2-e2677e34caca.png">

Reply:
Just did that. Let me know what you think!